packages/specification/prescriptions-api.yaml

```diff
@@ -0,0 +1,269 @@
+
+x-spec-publication:
+  operation-order:
+    - operations:
+        - method: POST
+          path: /GetMyPrescriptions/Bundle
+
+paths:
+  /GetMyPrescriptions/Bundle:
+    get:
+      operationId: get-my-prescriptions-bundle
+      summary: Get prescriptions for the given user
+      description: |
+        ## Overview
+        Use this endpoint to get a Bundle containing a list of the user's prescriptions.
+      parameters:
+        - $ref: "#/components/parameters/BearerAuthorisation"
+        - $ref: "#/components/parameters/NhsNumber"
+        - $ref: "#/components/parameters/RequestID"
+      responses:
+        "200":
+          description: Successful retrieval.
+          content:
+            application/fhir+json:
+              schema:
+                $ref: "#/components/schemas/bundle-response"
+              examples:
+                prescriptions-present:
+                  description: A successful response to a GetMyPrescriptions/Bundle request, containing one or more prescriptions.
+                  value:
+                    $ref: examples/GetMyPrescriptions/Bundle/success.json
+                prescriptions-not-present:
+                  description: A successful response to a GetMyPrescriptions/Bundle request, where no prescriptions were found.
+                  value:
+                    $ref: examples/GetMyPrescriptions/Bundle/success-empty.json
+        "4XX":
+          description: Invalid request.
+          content:
+            application/fhir+json:
+              schema:
+                $ref: "#/components/schemas/operation-outcome"
+              examples:
+                example:
+                  description: |
+                    An error response to a GetMyPrescriptions/Bundle request.
+                    Resource not found for the given NHS number.
+                  value:
+                    $ref: examples/GetMyPrescriptions/Bundle/error.json
+
+components:
+  parameters:
+    BearerAuthorisation:
+      in: header
+      name: Authorization
+      description: |
+        An [OAuth 2.0 bearer token](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#user-restricted-apis).
+      required: true
+      schema:
+        type: string
+        format: '^Bearer\ [[:ascii:]]+$'
+        example: "Bearer g1112R_ccQ1Ebbb4gtHBP1aaaNM"
+    NhsNumber:
+      in: header
+      name: NHS-Number
+      description: The NHS number of the patient whose prescriptions are to be returned.
+      required: true
+      schema:
+        type: string
+        example: "9434765919"
+    RequestID:
+      in: header
+      name: X-Request-ID
+      required: true
+      description: |
+        A globally unique identifier (GUID) for the request, which we use to de-duplicate repeated requests and to trace the request if you contact our helpdesk.
```

Comment:
Sorry I think the scenario for how we'd use this is different for this call. For normal Spine calls which are Async, so can appear to have failed we need client to resend and we de-dupe. For this it's simply an ID we can pass through the whole process to correlate the different log points in different components together.

Reply:
Reworded to relect this.